src/Storefront/Resources/views/storefront/component/product/listing.html.twig

```diff
@@ -22,10 +29,13 @@
 {% block product_listing %}
     <div class="cms-element-product-listing-wrapper"
          data-listing-pagination="true"
+        {% if feature('v6.7.0.0') %}
+         data-listing-pagination-options="{{ paginationConfig|json_encode }}"
+        {% else %}
          {# ludtwig-ignore html-string-quotation #}
          data-listing-pagination-options='{{ paginationConfig }}'
+        {% endif %}
          data-listing="true"
-         {# ludtwig-ignore html-string-quotation #}
          data-listing-options='{{ listingPagination|json_encode }}'>
```

Comment:
I just noticed I missed changing the quotes here
```suggestion
         data-listing-options="{{ listingPagination|json_encode }}">
```

Reply:
Thanks, I added this suggestion to the internal MR.